core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/pythonworker/promisehandlers/DebugCommandHandler.scala

```diff
@@ -0,0 +1,7 @@
+package edu.uci.ics.amber.engine.architecture.pythonworker.promisehandlers
+
+import edu.uci.ics.amber.engine.common.rpc.AsyncRPCServer.ControlCommand
+
+object DebugCommandHandler {
+  final case class DebugCommand(cmd: String) extends ControlCommand[Unit]
```

Comment:
seems like `PythonDebugCommand` is the command sent from client to controller, `DebugCommand` is from controller to worker, maybe we can call it `DebugCommandToWorker` to make the difference more explicit

Reply:
good point. actually I have been following another naming convention for `ConsoleMessage` and `PythonConsoleMessage`:
- ConsoleMessage refers to command between client and controller.
- PythonConsoleMessage refers to command between controller and python worker.

Do you think the above naming convention is good enough, or do you prefer to include Worker to make it explicit?